src/sp/transport/udp/udp.c

```diff
@@ -964,7 +965,17 @@ udp_pipe_recv(void *arg, nni_aio *aio)
 		nni_mtx_unlock(&ep->mtx);
 		return;
 	}
-
+	//If there are messages in the queue, retrieve the messages from the queue during reception 
+	//without waiting for the udp_recv_data operation to complete.
+	if(!nni_lmq_empty(&p->rx_mq)){
```

Comment:
```suggestion
	if (!nni_lmq_empty(&p->rx_mq)) {
```

clang-format will take care of this for you automatically as this project has a `.clang-format` file.